src/momento/internal/_utilities/_permissions.py

```diff
@@ -0,0 +1,225 @@
+from typing import Union
+
+from momento_wire_types import permissionmessages_pb2 as permissions_pb
+
+from momento.auth.access_control.disposable_token_scope import (
+    CacheItemKey,
+    CacheItemKeyPrefix,
+    DisposableTokenCachePermission,
+    DisposableTokenCachePermissions,
+    DisposableTokenScope,
+)
+from momento.auth.access_control.permission_scope import (
+    CacheName,
+    CachePermission,
+    CacheRole,
+    Permission,
+    Permissions,
+    PermissionScope,
+    PredefinedScope,
+    TopicName,
+    TopicPermission,
+    TopicRole,
+)
+from tests.utils import str_to_bytes
+
+
+class SuperuserPermissions(PredefinedScope):
+    pass
```

Comment:
This class is private in the JS sdk and I tried to make it private / internal-only here but it's still importable when trying it out from the examples directory. 
I can also just omit this class for now since it's not used for disposable tokens